src/applications/personalization/profile/components/direct-deposit/BankInfoCNP.jsx

```diff
@@ -160,7 +160,9 @@ export const DirectDepositCNP = ({
       </dl>
       <button
         className={classes.editButton}
-        aria-label={'Edit your direct deposit bank information'}
+        aria-label={
+          'Edit your direct deposit for disability compensation and pension benefits bank information'
```

Comment:
Did you mean to also update this copy in v1? Looks good to me though since it's more specific 👍

Reply:
I did because why not? :-)